Documentation/Setup/Config/Index.rst

```diff
@@ -1568,26 +1329,22 @@ noPageTitle
 pageRendererTemplateFile
 ------------------------
 
-.. container:: table-row
-
-   Property
-         pageRendererTemplateFile
+..  confval:: pageRendererTemplateFile
+    :name: config-pageRendererTemplateFile
+    :Data type: :ref:`data-type-string`
+    :Default: file:`EXT:core/Resources/Private/Templates/PageRenderer.html`
 
-   Data type
-         :ref:`data-type-string`
 
-   Default
-         :file:`EXT:core/Resources/Private/Templates/PageRenderer.html`
+    Sets the template for page renderer class
+    :php:`TYPO3\CMS\Core\Page\PageRenderer`.
 
-   Description
-         Sets the template for page renderer class
-         :php:`TYPO3\CMS\Core\Page\PageRenderer`.
+Example
+~~~~~~~
 
-   Example
-         .. code-block:: typoscript
-            :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
 
-            pageRendererTemplateFile = fileadmin/test_pagerender.html
+    pageRendererTemplateFile = EXT:my_extension/Ressources/Public/Templates/TestPagerender.html
```

Comment:
```suggestion
    pageRendererTemplateFile = EXT:my_extension/Ressources/Private/Templates/TestPagerender.html
```

This should not be exposed to the outside as this is used internally by PageRenderer.